src/main/resources/META-INF/resources/index.html

```diff
@@ -70,6 +70,19 @@
     </style>
     <script>
       $(function() {
+
+        // wrapper around jQuery.ajax to be able to POST JSON data
+        $.postJSON = function(url, data, callback) {
+          return jQuery.ajax({
```

Comment:
I think just `$.ajax` should work. I don't really know the difference a shorthand.